include/openthread/border_agent.h

```diff
@@ -117,6 +117,17 @@ typedef struct otBorderAgentSessionIterator
     uint64_t mData;
 } otBorderAgentSessionIterator;
 
+#define OT_BORDER_AGENT_MESHCOP_SERVICE_TXT_DATA_MAX_LENGTH 128
```

Comment:
What happens if the TXT data exceeds this length?

Reply:
Add to the API documentation?

I noticed that we have `...` in many places when documenting errors. We should try to avoid this and be more explicit about the set of possible errors and their meaning.

Reply:
Agree! I'll pay attention to the API return errors. 

Added it in API documentation.